website/moneybirdsynchronization/services.py

```diff
@@ -40,6 +57,54 @@ def create_or_update_contact(member):
     return moneybird_contact
 
 
+def synchronize_moneybird():
+    """Perform all synchronization to moneybird."""
+    logger.info("Starting moneybird synchronization.")
+
+    # First make sure all moneybird contacts' SEPA mandates are up to date.
+    sync_contacts_with_outdated_mandates()
```

Comment:
I would refactor this to simply sync_contacts (but can be done in the future as well)

Reply:
Yup I though about that as well, will probably do that at some point but for now it doesn't really matter